sympy/physics/vector/frame.py

```diff
@@ -53,6 +53,8 @@ def __new__(cls, name, frame, index):
         assumptions = {}
         super()._sanitize(assumptions, cls)
         obj = super().__xnew__(cls, name, **assumptions)
+        if isinstance(frame, Body):
```

Comment:
Body is not defined here too. Add the same line at the start of this file too with the other import statements.

Reply:
I did add this line but it is giving another error
```
ImportError: cannot import name 'CoordinateSym' from partially initialized module 'sympy.physics.vector' (most likely due to a circular import) (/home/psychopirate/Documents/sym/mj/sympy/sympy/physics/vector/__init__.py)
```

Reply:
I added the line and it is giving me this error
```
ImportError: cannot import name 'CoordinateSym' from partially initialized module 'sympy.physics.vector' (most likely due to a circular import) (/home/mj/Documents/sym/mj/sympy/sympy/physics/vector/__init__.py)
```

Reply:
I see import Body may have caused a circular import. This can be solved by removing import body from the top of the document.  Instead of at the top, we need to import body whenever it is required i.e. just before every `ifinstance(****, Body)` statement. Something like this but before every `ifinstance` case.
```python
   from sympy.physics.mechanics import Body  
    if isinstance(otherframe, Body):
        otherframe = Body.frame
```
Please note to use `from sympy.physics.mechanics` instead of `from sympy.physics.mechanics.body`. Also do the same for the other file too. I have tested these changes and the tests run fine.

Reply:
Thank you so much! @mostlyaman